calico-enterprise/operations/cnx/configure-identity-provider.mdx

```diff
@@ -248,6 +248,7 @@ When configuring your cluster, you may be asked for the following inputs:
      bindPW: <your-base64-bind-password>
      rootCA: <your-base64-ca-cert-pem>
    ```
+###
```

Comment:
Do we need this change?

Reply:
It looks like a variable where user inserts their own value.